MSMAll/MSMAllPipeline.sh

```diff
@@ -138,75 +147,177 @@ log_File_Must_Exist "${TopographyTarget}"
 output_proc_string="_vn" #To VN only to indicate that we did not revert the bias field before computing VN
 log_Msg "output_proc_string: ${output_proc_string}"
 
-expected_concatenated_output_dir="${StudyFolder}/${Subject}/MNINonLinear/Results/${OutputfMRIName}"
-expected_concatenated_output_file="${expected_concatenated_output_dir}/${OutputfMRIName}${fMRIProcSTRING}${output_proc_string}.dtseries.nii"
-before_vn_output_file="${expected_concatenated_output_dir}/${OutputfMRIName}${fMRIProcSTRING}_novn.dtseries.nii"
+IsLongitudinal=$(opts_StringToBool "$IsLongitudinal")
+
+if (( IsLongitudinal ));  then 
+    
+    if [ -z "$SubjectLong" -o -z "$Sessions" -o -z "TemplateLong" -o -z "${mrfixNamesToUse}" -o -z "$mrfixConcatName" ]; then 
```

Comment:
```suggestion
    if [[ -z "$SubjectLong" || -z "$Sessions" || -z "TemplateLong" || -z "${mrfixNamesToUse}" || -z "$mrfixConcatName" ]]; then 
```
Same comment as the other PR.